## Part 3: AI Labeling, Journal-Topic Analysis & Methodology-Topic Analysis

**Project:** Management Accounting Research — Hot Topics  
**Input:** `papers_with_topics_revised.csv`

This notebook covers three analytical components added in response to reviewer feedback:
1. **AI-assisted topic labeling** — assigning human-readable labels to the 13 BERTopic clusters
2. **Journal-topic distribution** — does topic prevalence vary across MAR, JMAR, and JOMAC?
3. **Methodology classification** — do certain methodologies cluster within specific topics?

---

| Output File | Description |
|---|---|
| `papers_with_labels.csv` | Dataset with AI labels and methodology classifications |
| `topic_journal_crosstab.csv` | Cross-tab: topics × journals |
| `fig_journal_topic.png` | Stacked bar: journal distribution per topic |
| `fig_methodology_topic.png` | Stacked bar: methodology distribution per topic |


In [ ]:
import os, re, warnings
import numpy as np
import pandas as pd
import matplotlib
matplotlib.use('Agg')
import matplotlib.pyplot as plt
import seaborn as sns
warnings.filterwarnings('ignore')
sns.set_theme(style='whitegrid', palette='muted')
plt.rcParams['figure.dpi'] = 150
print('Imports successful.')

Imports successful.


## Step 1 — Load Data

In [ ]:
df = pd.read_csv("papers_with_topics.csv", encoding="utf-8-sig")
df["topic_id"] = df["topic_id"].astype(int)

JOURNAL_MAP = {
    "Management Accounting Research":               "MAR",
    "Journal of Management Accounting Research":    "JMAR",
    "Journal of Management Control":                "JOMAC",
}
df["journal"] = df["Source title"].map(JOURNAL_MAP)

print(f"Corpus: {len(df):,} papers")
print(dict(df["journal"].value_counts()))
print(f"Named topics (original BERTopic): {df[df['topic_id']!=-1]['topic_id'].nunique()}")
print(f"Noise papers: {(df['topic_id']==-1).sum()} ({(df['topic_id']==-1).sum()/len(df)*100:.1f}%)")


Corpus: 314 papers
{'JMAR': np.int64(143), 'JOMAC': np.int64(88), 'MAR': np.int64(83)}
Named topics (original BERTopic): 11
Noise papers: 40 (12.7%)


## Step 2 — AI-Assisted Topic Labeling

Labels were derived via a hybrid human–AI coding procedure. For each topic, the top-10 c-TF-IDF keywords and the titles and first 300 characters of the three highest-probability abstracts were provided to the model. All labels were validated by the authors against the full paper list for each cluster.

**Prompt template (verbatim):**

> "You are an expert in management accounting research with deep familiarity with the journals Management Accounting Research (MAR), Journal of Management Accounting Research (JMAR), and Journal of Management Control (JMC). Based on the top keywords and representative abstracts below, provide: (1) a concise topic label (4–7 words), (2) a 2–3 sentence description of the topic's thematic core, (3) the primary management accounting subfield, grounded in the subfield taxonomy of Hopper and Bui (2015) (e.g., management control systems, performance measurement, cost management, sustainability, strategic management accounting), and (4) a flag indicating whether the cluster overlaps substantially with another identified topic, or whether its thematic core falls primarily outside the core scope of management accounting research (e.g., financial accounting, corporate governance without direct MA relevance)."


In [3]:
AI_LABELS = {
    0:  "Executive Compensation, Contracting & Governance",
    1:  "Performance Measurement & Behavioral Management Control",
    2:  "MA Profession, Role & Digital Transformation",
    3:  "Pay Transparency & Compensation Dispersion",
    4:  "Cost Behavior & Supply Chain Management",
    5:  "Management Control Systems & Organizational Risk",
    6:  "Sustainability & Environmental Management Control",
    7:  "Budgeting Practices & Creative Performance",
    8:  "Subjective Performance Evaluation & Procedural Fairness",
    9:  "Trust & Relational Management Control",
    10: "Analytical Models of Optimal Contracting & Allocation",
}

df["topic_label"] = df["topic_id"].map(AI_LABELS).fillna("[noise]")

print("Topic labels assigned:")
label_counts = (
    df[df["topic_id"] != -1]
    .groupby(["topic_id", "topic_label"])
    .size()
    .reset_index(name="n")
    .sort_values("n", ascending=False)
)
display(label_counts)

df.to_csv("papers_with_labels.csv", index=False, encoding="utf-8-sig")
print(f"Saved papers_with_labels.csv ({len(df)} rows)")


Topic labels assigned:


,topic_id,topic_label,n
0,0,"Executive Compensation, Contracting & Governance",60
1,1,Performance Measurement & Behavioral Managemen...,51
2,2,"MA Profession, Role & Digital Transformation",37
3,3,Pay Transparency & Compensation Dispersion,25
4,4,Cost Behavior & Supply Chain Management,24
5,5,Management Control Systems & Organizational Risk,22
6,6,Sustainability & Environmental Management Control,17
7,7,Budgeting Practices & Creative Performance,11
8,8,Subjective Performance Evaluation & Procedural...,10
9,9,Trust & Relational Management Control,9


Saved papers_with_labels.csv (314 rows)


## Step 3 — Journal-Topic Distribution Analysis

For each named topic, the paper counts by journal (MAR, JMAR, JOMAC) are cross-tabulated. A chi-square test of independence checks whether topic prevalence differs significantly across journals.

In [7]:
SHORT_LABELS = {
    0:  "T0: Exec. Comp. & Governance",
    1:  "T1: Perf. Meas. & Behavior",
    2:  "T2: MA Profession & Digital",
    3:  "T3: Pay Transparency",
    4:  "T4: Cost Behavior",
    5:  "T5: MCS & Risk",
    6:  "T6: Sustainability",
    7:  "T7: Budgeting",
    8:  "T8: Subj. Perf. Eval.",
    9:  "T9: Trust & Relational",
    10: "T10: Analytical Models",
}

df_named = df[df["topic_id"] != -1].copy()

# Counts cross-tab
ct = pd.crosstab(df_named["topic_id"], df_named["journal"])
ct["Total"] = ct.sum(axis=1)
ct["Label"] = ct.index.map(SHORT_LABELS)
ct = ct.sort_values("Total", ascending=False)
ct.to_csv("topic_journal_crosstab.csv")
print("Counts:")
display(ct[["Label","MAR","JMAR","JOMAC","Total"]])

# Percentage table
ct_pct = ct[["MAR","JMAR","JOMAC"]].div(ct["Total"], axis=0).mul(100).round(1)
ct_pct["Label"] = ct["Label"]
print("\nRow percentages (% within topic):")
display(ct_pct[["Label","MAR","JMAR","JOMAC"]])

# ── Chi-square test (manual calculation without scipy) ────────────────────
obs = ct[["MAR","JMAR","JOMAC"]].values.astype(float)
row_totals = obs.sum(axis=1, keepdims=True)
col_totals = obs.sum(axis=0, keepdims=True)
grand_total = obs.sum()
expected = row_totals @ col_totals / grand_total

chi2 = float(((obs - expected)**2 / expected).sum())
dof  = (obs.shape[0] - 1) * (obs.shape[1] - 1)  # (11-1)*(3-1) = 20

print(f"\n--- Chi-square test: topic × journal ---")
print(f"chi2 = {chi2:.2f}, dof = {dof}")
print("(p-value requires scipy; by convention chi2({dof})>{round(chi2,1)} >> critical value => highly significant)")

# Cramér's V
import math
n = grand_total
k = min(obs.shape) - 1
cramers_v = math.sqrt(chi2 / (n * k))
print(f"Cramér's V = {cramers_v:.3f}")


Counts:


journal,Label,MAR,JMAR,JOMAC,Total
topic_id,,,,,
0,T0: Exec. Comp. & Governance,7,43,10,60
1,T1: Perf. Meas. & Behavior,15,29,7,51
2,T2: MA Profession & Digital,13,12,12,37
3,T3: Pay Transparency,8,9,8,25
4,T4: Cost Behavior,4,14,6,24
5,T5: MCS & Risk,3,4,15,22
6,T6: Sustainability,7,2,8,17
7,T7: Budgeting,1,3,7,11
8,T8: Subj. Perf. Eval.,0,7,3,10



Row percentages (% within topic):


journal,Label,MAR,JMAR,JOMAC
topic_id,,,,
0,T0: Exec. Comp. & Governance,11.7,71.7,16.7
1,T1: Perf. Meas. & Behavior,29.4,56.9,13.7
2,T2: MA Profession & Digital,35.1,32.4,32.4
3,T3: Pay Transparency,32.0,36.0,32.0
4,T4: Cost Behavior,16.7,58.3,25.0
5,T5: MCS & Risk,13.6,18.2,68.2
6,T6: Sustainability,41.2,11.8,47.1
7,T7: Budgeting,9.1,27.3,63.6
8,T8: Subj. Perf. Eval.,0.0,70.0,30.0



--- Chi-square test: topic × journal ---
chi2 = 66.14, dof = 20
(p-value requires scipy; by convention chi2({dof})>{round(chi2,1)} >> critical value => highly significant)
Cramér's V = 0.347


In [8]:
import scipy

## Step 4 — Methodology Classification

A keyword-based classifier applied to each abstract identifies the primary research methodology. Five substantive categories are distinguished — **Experimental**, **Survey/Field**, **Qualitative**, **Analytical** (formal theory), and **Archival** — with residual papers classified as **Other/Mixed**.

> **Note on validity:** Abstract-based classification is inherently approximate; method descriptions vary across journals and authors. Results should be interpreted as indicative of broad methodological tendencies rather than precise counts.

In [9]:
def classify_methodology(abstract):
    if not isinstance(abstract, str): return "Other/Mixed"
    t = abstract.lower()
    s = {
        "Experimental": sum([
            bool(re.search(r"\bexperiment(al|s|ally)?\b", t)),
            bool(re.search(r"\blab(oratory)?\b", t)),
            bool(re.search(r"\bparticipant(s)?\b", t)),
            bool(re.search(r"\bvignette\b", t)),
            bool(re.search(r"\bmanipulat(e|ed|ion)\b", t)),
        ]),
        "Analytical": sum([
            bool(re.search(r"\banalytic(al|ally)?\b", t)),
            bool(re.search(r"\boptim(al|iz|ization)\b", t)),
            bool(re.search(r"\bequilibrium\b", t)),
            bool(re.search(r"\bgame.theor\b", t)),
            bool(re.search(r"\bproposition\b", t)),
        ]),
        "Qualitative": sum([
            bool(re.search(r"\bcase stud(y|ies)\b", t)),
            bool(re.search(r"\binterview(s|ed|ing)\b", t)),
            bool(re.search(r"\bqualitative\b", t)),
            bool(re.search(r"\binterpretive\b", t)),
            bool(re.search(r"\bethnograph\b", t)),
        ]),
        "Survey/Field": sum([
            bool(re.search(r"\bsurvey(s|ed|ing)?\b", t)),
            bool(re.search(r"\bquestionnaire\b", t)),
            bool(re.search(r"\bresponse rate\b", t)),
            bool(re.search(r"\bfield stud(y|ies)\b", t)),
        ]),
        "Archival": sum([
            bool(re.search(r"\barchival\b", t)),
            bool(re.search(r"\bcompustat\b", t)),
            bool(re.search(r"\bpanel data\b", t)),
            bool(re.search(r"\bfirm.year\b", t)),
        ]),
    }
    best = max(s, key=s.get)
    return best if s[best] > 0 else "Other/Mixed"

df["methodology"] = df["Abstract"].apply(classify_methodology)
df.to_csv("papers_with_labels.csv", index=False)
print("Methodology distribution (full corpus):")
display(df["methodology"].value_counts().rename("n").to_frame())
print("\n✓ papers_with_labels.csv updated")

Methodology distribution (full corpus):


,n
methodology,
Other/Mixed,134
Experimental,80
Survey/Field,41
Qualitative,35
Analytical,18
Archival,6



✓ papers_with_labels.csv updated


In [10]:
topic_col = "topic_id" if "topic_id" in df.columns else "topic"

methodology_by_topic_pct = (
    df.groupby([topic_col, "methodology"])
      .size()
      .unstack(fill_value=0)
      .pipe(lambda x: x.div(x.sum(axis=1), axis=0) * 100)
      .round(1)
)

print("Methodology shares by topic (%):")
display(methodology_by_topic_pct)

Methodology shares by topic (%):


methodology,Analytical,Archival,Experimental,Other/Mixed,Qualitative,Survey/Field
topic_id,,,,,,
-1,0.0,2.5,15.0,47.5,22.5,12.5
0,6.7,1.7,13.3,70.0,1.7,6.7
1,3.9,0.0,64.7,17.6,2.0,11.8
2,5.4,0.0,5.4,37.8,29.7,21.6
3,8.0,0.0,64.0,20.0,4.0,4.0
4,4.2,8.3,8.3,75.0,4.2,0.0
5,0.0,0.0,9.1,36.4,36.4,18.2
6,17.6,5.9,5.9,29.4,11.8,29.4
7,9.1,0.0,0.0,45.5,0.0,45.5


In [11]:
# ── Split figure into two separate saved plots ─────────────────────────────────

# Prepare sorted data
ct_plot = ct.sort_values("Total", ascending=True)

if "Label" not in ct_plot.columns:
    ct_plot["Label"] = ct_plot.index.map(SHORT_LABELS)

journals_list = ["MAR", "JMAR", "JOMAC"]
colors_j = ["#1a6b3a", "#5ab284", "#aed8bd"]

# ── (a) Stacked bar chart ──────────────────────────────────────────────────────
fig1, ax = plt.subplots(figsize=(8, 8))

bottoms = np.zeros(len(ct_plot))

for j, col in enumerate(journals_list):
    vals = ct_plot[col].values
    ax.barh(
        ct_plot["Label"],
        vals,
        left=bottoms,
        color=colors_j[j],
        label=col,
        height=0.65
    )

    for i, (v, b) in enumerate(zip(vals, bottoms)):
        if v >= 2:
            ax.text(
                b + v / 2,
                i,
                str(v),
                ha="center",
                va="center",
                fontsize=8.5,
                color="white",
                fontweight="bold"
            )

    bottoms += vals

ax.set_xlabel("Number of Papers", fontsize=10)
ax.set_title("Topic Distribution by Journal", fontsize=11, fontweight="bold")
ax.legend(title="Journal", fontsize=9)
ax.spines["top"].set_visible(False)
ax.spines["right"].set_visible(False)
ax.tick_params(axis="y", labelsize=9)

plt.tight_layout()
plt.savefig("fig_journal_topic_bar.png", dpi=150, bbox_inches="tight")
plt.show()

print("✓ fig_journal_topic_bar.png saved")


# ── (b) Heatmap ────────────────────────────────────────────────────────────────
ct_pct_h = (
    ct[["MAR", "JMAR", "JOMAC"]]
    .div(ct["Total"], axis=0)
    .mul(100)
    .round(1)
)

ct_pct_h.index = ct["Label"]
ct_pct_h = ct_pct_h.loc[ct_plot["Label"]]

fig2, ax2 = plt.subplots(figsize=(7, 8))

sns.heatmap(
    ct_pct_h,
    annot=True,
    fmt=".0f",
    cmap="Greens",
    linewidths=0.5,
    linecolor="white",
    ax=ax2,
    cbar_kws={"label": "% within topic"},
    vmin=0,
    vmax=85,
    annot_kws={"size": 9}
)

ax2.set_title("Journal Share per Topic (%)", fontsize=11, fontweight="bold")
ax2.set_xlabel("")
ax2.set_ylabel("")

plt.tight_layout()
plt.savefig("fig_journal_topic_heatmap.png", dpi=150, bbox_inches="tight")
plt.show()

print("✓ fig_journal_topic_heatmap.png saved")

✓ fig_journal_topic_bar.png saved
✓ fig_journal_topic_heatmap.png saved


In [12]:
# ── Additional outputs: journal baselines, journal × methodology, exact p-value ──

import numpy as np
from scipy.stats import chi2_contingency

# Keep only the three focal journals if needed
JOURNALS = ["MAR", "JMAR", "JOMAC"]

# =============================================================================
# 1) Overall paper counts by journal across the full corpus
# =============================================================================
journal_counts_full = (
    df["journal"]
      .value_counts()
      .reindex(JOURNALS, fill_value=0)
      .rename_axis("journal")
      .reset_index(name="n")
)
journal_counts_full["share_pct"] = (
    journal_counts_full["n"] / journal_counts_full["n"].sum() * 100
).round(1)

journal_counts_full.to_csv("journal_counts_full_corpus.csv", index=False)
print("Overall paper counts by journal (full corpus):")
display(journal_counts_full)

# Optional: baseline shares as a dict for later comparison
journal_baseline = (
    journal_counts_full.set_index("journal")["share_pct"] / 100
).to_dict()

# =============================================================================
# 2) Topic × journal overrepresentation relative to journal baseline
#    (= within-topic journal share / full-corpus journal share)
# =============================================================================
df_named = df[df["topic_id"] != -1].copy()

topic_journal_counts = pd.crosstab(df_named["topic_id"], df_named["journal"]).reindex(
    columns=JOURNALS, fill_value=0
)
topic_journal_counts["Total"] = topic_journal_counts.sum(axis=1)
topic_journal_counts["Label"] = topic_journal_counts.index.map(SHORT_LABELS)

topic_journal_pct = (
    topic_journal_counts[JOURNALS]
    .div(topic_journal_counts["Total"], axis=0)
    .mul(100)
    .round(1)
)

topic_journal_lift = (
    topic_journal_counts[JOURNALS]
    .div(topic_journal_counts["Total"], axis=0)
    .div(pd.Series(journal_baseline), axis=1)
    .round(2)
)
topic_journal_lift["Label"] = topic_journal_counts["Label"]

topic_journal_pct_out = topic_journal_pct.copy()
topic_journal_pct_out["Label"] = topic_journal_counts["Label"]

topic_journal_pct_out.to_csv("topic_journal_row_percentages.csv")
topic_journal_lift.to_csv("topic_journal_overrepresentation_lift.csv")

print("\nTopic × journal row percentages (% within topic):")
display(topic_journal_pct_out[["Label", "MAR", "JMAR", "JOMAC"]])

print("\nTopic × journal overrepresentation vs full-corpus journal baseline (lift > 1 = overrepresented):")
display(topic_journal_lift[["Label", "MAR", "JMAR", "JOMAC"]])

# =============================================================================
# 3) Exact chi-square test for topic × journal using scipy
# =============================================================================
obs_topic_journal = topic_journal_counts[JOURNALS].values

chi2, p_value, dof, expected = chi2_contingency(obs_topic_journal, correction=False)
cramers_v = np.sqrt(chi2 / (obs_topic_journal.sum() * (min(obs_topic_journal.shape) - 1)))

expected_df = pd.DataFrame(
    expected,
    index=topic_journal_counts.index,
    columns=JOURNALS
)
expected_df["Label"] = topic_journal_counts["Label"]
expected_df.to_csv("topic_journal_expected_counts.csv")

print("\n--- Chi-square test: topic × journal ---")
print(f"chi2 = {chi2:.4f}")
print(f"dof = {dof}")
print(f"p-value = {p_value:.10f}")
print(f"Cramér's V = {cramers_v:.3f}")

# =============================================================================
# 4) Journal × methodology cross-tab
# =============================================================================
journal_method_counts = pd.crosstab(df["journal"], df["methodology"]).reindex(
    index=JOURNALS, fill_value=0
)

journal_method_row_pct = (
    journal_method_counts
    .div(journal_method_counts.sum(axis=1), axis=0)
    .mul(100)
    .round(1)
)

journal_method_counts.to_csv("journal_methodology_crosstab_counts.csv")
journal_method_row_pct.to_csv("journal_methodology_crosstab_rowpct.csv")

print("\nJournal × methodology counts:")
display(journal_method_counts)

print("\nJournal × methodology row percentages (% within journal):")
display(journal_method_row_pct)

# Optional: inferential test for journal × methodology
chi2_jm, p_jm, dof_jm, expected_jm = chi2_contingency(journal_method_counts.values, correction=False)
cramers_v_jm = np.sqrt(
    chi2_jm / (journal_method_counts.values.sum() * (min(journal_method_counts.shape) - 1))
)

print("\n--- Chi-square test: journal × methodology ---")
print(f"chi2 = {chi2_jm:.4f}")
print(f"dof = {dof_jm}")
print(f"p-value = {p_jm:.10f}")
print(f"Cramér's V = {cramers_v_jm:.3f}")

Overall paper counts by journal (full corpus):


,journal,n,share_pct
0,MAR,83,26.4
1,JMAR,143,45.5
2,JOMAC,88,28.0



Topic × journal row percentages (% within topic):


journal,Label,MAR,JMAR,JOMAC
topic_id,,,,
0,T0: Exec. Comp. & Governance,11.7,71.7,16.7
1,T1: Perf. Meas. & Behavior,29.4,56.9,13.7
2,T2: MA Profession & Digital,35.1,32.4,32.4
3,T3: Pay Transparency,32.0,36.0,32.0
4,T4: Cost Behavior,16.7,58.3,25.0
5,T5: MCS & Risk,13.6,18.2,68.2
6,T6: Sustainability,41.2,11.8,47.1
7,T7: Budgeting,9.1,27.3,63.6
8,T8: Subj. Perf. Eval.,0.0,70.0,30.0



Topic × journal overrepresentation vs full-corpus journal baseline (lift > 1 = overrepresented):


journal,Label,MAR,JMAR,JOMAC
topic_id,,,,
0,T0: Exec. Comp. & Governance,0.44,1.58,0.60
1,T1: Perf. Meas. & Behavior,1.11,1.25,0.49
2,T2: MA Profession & Digital,1.33,0.71,1.16
3,T3: Pay Transparency,1.21,0.79,1.14
4,T4: Cost Behavior,0.63,1.28,0.89
5,T5: MCS & Risk,0.52,0.40,2.44
6,T6: Sustainability,1.56,0.26,1.68
7,T7: Budgeting,0.34,0.60,2.27
8,T8: Subj. Perf. Eval.,0.00,1.54,1.07



--- Chi-square test: topic × journal ---
chi2 = 66.1405
dof = 20
p-value = 0.0000007671
Cramér's V = 0.347

Journal × methodology counts:


methodology,Analytical,Archival,Experimental,Other/Mixed,Qualitative,Survey/Field
journal,,,,,,
MAR,5,1,18,31,13,15
JMAR,7,2,46,77,5,6
JOMAC,6,3,16,26,17,20



Journal × methodology row percentages (% within journal):


methodology,Analytical,Archival,Experimental,Other/Mixed,Qualitative,Survey/Field
journal,,,,,,
MAR,6.0,1.2,21.7,37.3,15.7,18.1
JMAR,4.9,1.4,32.2,53.8,3.5,4.2
JOMAC,6.8,3.4,18.2,29.5,19.3,22.7



--- Chi-square test: journal × methodology ---
chi2 = 45.7210
dof = 10
p-value = 0.0000016111
Cramér's V = 0.270


In [19]:
fig2, ax = plt.subplots(figsize=(8, 6))

# Ensure numeric
jm_plot = journal_method_counts.apply(pd.to_numeric, errors="coerce").fillna(0).copy()
jm_plot["Label"] = jm_plot.index

method_list = [col for col in jm_plot.columns if col != "Label"]

bottoms = np.zeros(len(jm_plot))

# Use SAME palette logic as your topic chart
colors_m = sns.color_palette("Greens_d", len(method_list))

for j, col in enumerate(method_list):
    vals = jm_plot[col].values.astype(float)

    ax.barh(
        jm_plot["Label"],
        vals,
        left=bottoms,
        color=colors_m[j],
        label=col,
        height=0.65
    )

    for i, (v, b) in enumerate(zip(vals, bottoms)):
        if v >= 2:
            ax.text(
                b + v / 2,
                i,
                str(int(v)),
                ha="center",
                va="center",
                fontsize=8.5,
                color="white",
                fontweight="bold"
            )

    bottoms += vals

ax.set_xlabel("Number of Papers", fontsize=10)
ax.set_title("Methodology Distribution by Journal", fontsize=11, fontweight="bold")
ax.legend(title="Methodology", fontsize=9)

ax.spines["top"].set_visible(False)
ax.spines["right"].set_visible(False)
ax.tick_params(axis="y", labelsize=9)

plt.tight_layout()
plt.savefig("fig_journal_methodology_bar.png", dpi=150, bbox_inches="tight")
plt.show()

## Step 5 — Key Findings Summary

### Journal-Topic Concentrations

Five topics exhibit a dominant journal share of ≥ 60% (based on original 11 BERTopic clusters):

| Topic | Dominant Journal | Share |
|---|---|---|
| T0: Exec. Comp. & Governance | JMAR | 72% |
| T8: Subj. Perf. Eval. | JMAR | 70% |
| T5: MCS & Risk | JOMAC | 68% |
| T7: Budgeting | JOMAC | 64% |
| T1: Perf. Meas. & Behavior | JMAR | 57% |

The chi-square test confirms that this distribution is highly non-random (χ²(20), p < 0.001). JMAR concentrates on behavioral/incentive topics; JOMAC on organizational/control systems; MAR shows the broadest thematic profile.

Topics 0 and 2 are semantically broad clusters: Topic 0 spans executive compensation and governance mechanisms (72% JMAR); Topic 2 covers both the professional role of management accountants and digitalization of the finance function (evenly distributed across all three journals).

### Methodology-Topic Associations

Topics with a distinctly experimental methodology profile: T1 (Perf. Meas. & Behavior), T3 (Pay Transparency), T8 (Subj. Perf. Eval.). Topics 0 and 2 show heterogeneous methodology profiles (predominantly Other/Mixed), consistent with their status as broad clusters.
